ui/src/locales/en-US.json

```diff
@@ -113,6 +113,7 @@
   "common.diff.tips": "The following is a before and after comparison of this change.",
   "common.flow.explain": "Explain",
   "common.know.text": "I see",
+  "common.know.not.me.text": "Got it",
```

Comment:
Why not use I see?

Reply:
A little difference，can be changed to “I  See”